app/questionnaire/placeholder_transforms.py

```diff
@@ -80,13 +80,27 @@ def format_number(self, number):
         return ""
 
     @staticmethod
-    def calculate_years_difference(first_date, second_date):
-        return str(
-            relativedelta(
-                PlaceholderTransforms.parse_date(second_date),
-                PlaceholderTransforms.parse_date(first_date),
-            ).years
+    def calculate_difference(first_date, second_date):
```

Comment:
@petechd Should the method name be more descriptive?. Possibly `calculate_time_difference` or something along those lines?
@ajmaddaford thoughts?

Reply:
It's not the most accurate method to calculate time difference as it will never return amount of years and months or months and days in one string, only one of two, so it's best suited to calculate age?!

Reply:
How about `calculate_date_difference`?

I think we'll come back to this when we know more about the requirements for date formatting, for example we may want to work out the difference in years and months.